include/bluetooth/uuid.h

```diff
@@ -30,6 +30,15 @@ enum {
 	BT_UUID_TYPE_128,
 };
 
+/** Size in octets of a 16-bit UUID */
+#define BT_UUID16_SIZE                  2
```

Comment:
The namespace used by all existing macros in this header file is `BT_UUID_*`, whereas you're now introducing `BT_UUID16_*` etc. So for consistness, please make that `BT_UUID_16_*` etc.

Reply:
That's a good point. Actually to be consist it should be BT_UUID_SIZE_16.
Examples: BT_UUID_INIT_16, BT_UUID_DECLARE_32, BT_UUID_TYPE_128, etc.

Reply:
As you like it.  Changed.